contracts/OpenSTValueMock.sol

```diff
@@ -22,22 +22,32 @@ pragma solidity ^0.4.23;
 // ----------------------------------------------------------------------------
 
 import "./OpenSTValue.sol";
+import "./HasherMock.sol";
 
```

Comment:
@noslav please check once complete OpenSTValueMock.sol file wrt to natspec commenting guidelines.
The file is relatively new so we can update all if not aligned.

Reply:
Agreed!

Reply:
+1